tests/.infra/centos-ci/nightly/ocp-oauth-test.sh

```diff
@@ -34,14 +34,24 @@ installCheCtl
 deployCheIntoCluster  --che-operator-cr-patch-yaml=/tmp/custom-resource-patch.yaml
 seleniumTestsSetup
 
+export OPENSHIFT_USERNAME=developer
+export OPENSHIFT_PASSWORD=123
+export OPENSHIFT_REGULAR_USERNAME=developer
+export OPENSHIFT_REGULAR_PASSWORD=123
+export OPENSHIFT_REGULAR_EMAIL=${OPENSHIFT_REGULAR_USERNAME}@1.com
+
+export TEST_USER_NAME=admin
+export CHE_TESTUSER_NAME=${TEST_USER_NAME}
+export CHE_TESTUSER_PASSWORD=admin
+export CHE_TESTUSER_EMAIL=${TEST_USER_NAME}@admin.com
+
 bash tests/legacy-e2e/che-selenium-test/selenium-tests.sh \
-  --threads=1 \
+  --threads=3 \
   --host=${CHE_ROUTE} \
   --https \
   --port=443 \
   --multiuser \
   --fail-script-on-failed-tests \
-  --test=org.eclipse.che.selenium.site.ocpoauth.** \
```

Comment:
`org.eclipse.che.selenium.site.ocpoauth.**` tests are not being run by default. 

Did you ensure they were running after this change?